mlrun/runtimes/base.py

```diff
@@ -963,15 +963,17 @@ def _update_run_state(
             except KeyError:
                 updates = BaseRuntimeHandler._get_run_completion_updates(resp)
 
+        uid = get_in(resp, "metadata.uid")
         logger.debug(
             "Run updates",
+            name=get_in(resp, "metadata.name"),
+            uid=uid,
             kind=kind,
```

Comment:
what has changed that those changes are required? please add to pr description

Reply:
wanted to log them to make the log more informative (to understand which function is in context)